### PR TITLE
Merge in the database creation scripts and new README.md 

### PR DESCRIPTION
**Pull request by sarahquinn-kainos**
# Database Creation 

* created courses DB and Course table.
* populated Course with test data.
* modified README.md to show creation of a new user

### CODE REVIEW

**Review by Christopher154 — DISMISSED** [review]
👌

File: Kainos-map/SQL scripts/initialise_db.sql [hunk 1]
```diff
@@ -0,0 +1,21 @@
+DROP DATABASE IF EXISTS courses;
+CREATE DATABASE IF NOT EXISTS courses;
+USE courses;
+
+DROP TABLE IF EXISTS Course;
```
Comment (matpmag) [inline Kainos-map/SQL scripts/initialise_db.sql]: If the database has just been dropped, do we need this?

**Review by matpmag — CHANGES_REQUESTED** [review]

File: Kainos-map/SQL scripts/initialise_db.sql (hunk 1 above)
Comment (sarahquinn-kainos) [reply Kainos-map/SQL scripts/initialise_db.sql]: fair enoof

Comment (matpmag) [reply Kainos-map/SQL scripts/initialise_db.sql]: 🏖

**Review by ConorLeckey — APPROVED** [review]
lit

**Review by matpmag — APPROVED** [review]

**Pull request merged**